src/Apcu/bootstrap_zend.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Bootstrap for Zend Server Data Cache
+ */
+if (!function_exists('apc_add')) {
+    function apcu_add($key, $var = null, $ttl = 0) {
+        if (is_array($key)) {
+            $errors = array ();
+            foreach ($key as $cache_key => $cache_value) {
+                if (!apc_add ($cache_key, $cache_value, $ttl)) {
+                    $errors[] = $cache_key;
+                }
+            }
+            return $errors;
+        } else {
```

Comment:
brackets on if are standards
extra else is not needed, producing larger bytecode and so on